code/modules/spells/targeted/frenchcurse.dm

```diff
@@ -0,0 +1,39 @@
+/spell/targeted/frenchcurse
+	name = "French Curse"
+	desc = "This curse will silence your target for a very long time."
+
+	school = "evocation"
+	charge_max = 600
+	invocation = "FU'K Y'U D'NY"
+	invocation_type = "shout"
+	range = 1
+	cooldown_min = 100 //100 deciseconds reduction per rank
+
+	sparks_spread = 1
+	sparks_amt = 4
+
+	compatible_mobs = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
```

Comment:
You need to remove /mob/living/carbon/monkey from the compatible_mobs list since you only check for humans.